editor/tabbed_editor.go

```diff
@@ -139,6 +141,20 @@ func (e *TabbedEditor) CloseCurrentEditor() (name string, editor *CodeEditor) {
 	return "", nil
 }
 
+func (e *TabbedEditor) SaveAll() {
+
+	for name, editor := range e.editors {
+		f, err := os.Create(name)
+		if err != nil {
+			log.Printf("Could not save %s : %s", name, err)
+		}
+		defer f.Close()
+		if _, err := f.WriteString(editor.Text()); err != nil {
+			log.Printf("Could not write to file %s: %s", name, err)
+		}
+	}
+}
+
```

Comment:
We can add a `SaveAll()` method to the `MultiEditor` interface, [here](https://github.com/nelsam/vidar/blob/master/editor/split_editor.go#L41-L51).  That would force both the `TabbedEditor` and `SplitEditor` to implement `SaveAll()`, which is really what we want.